soepy/simulate/simulate_auxiliary.py

```diff
@@ -80,8 +82,15 @@ def pyth_simulate(
 
         # Convert to init age of child
         child_init_age = np.where(kids_init_draw == 0, -1, 0)
-        # Add column to state space
-        initial_states_in_period = np.c_[initial_states_in_period, child_init_age]
+
+        # Draw presence of partner in the first period
+        # Discuss: Best way to implement random draws by education
+        partner_status_init_draw = np.full(initial_states_in_period.shape[0], 0)
```

Comment:
So everybody enters without partner?

Reply:
Not any more. This was so just to ensure replication of the old regression vault.